aim/storage/utils.py

```diff
@@ -1,3 +1,30 @@
+from copy import deepcopy
+
+
+class KeysIterator:
```

Comment:
what's the purpose of the Iterator classes? is it something to be used in next PRs?

Reply:
Yes! There will be fast Cython implementations for `ItemsIterator`s always yielding tuples of `(key, value)`s, and there is no point of optimizing them as well. `RocksDB` always retrieves values under the hood. Maybe we'll optimize `KeysIterator` after `RocksDB` does.